otherlibs/win32unix/stat.c

```diff
@@ -15,6 +15,7 @@
 #define CAML_INTERNALS
 
 #include <errno.h>
+#include <float.h>
```

Comment:
```suggestion
/* Visual Studio 2013+ has nextafter; mingw-w64 always defines it */
#if !defined(__MINGW32__) && _MSC_VER < 1800
#include <float.h>
#define nextafter _nextafter
#else
#include <math.h>
#endif
```

Reply:
I tried to copy what's done in runtime/floats.c:
https://github.com/ocaml/ocaml/blob/ec24f98bec9c62da5f7e1cd1b1495fbd0f0c50e8/runtime/floats.c#L65-L76
I got Mingw wrong, but I don't see the `_MSC_VER < 1800` case in runtime/floats.c.  Could we try to have the same logic in both files?